README.md

```diff
@@ -7,7 +7,7 @@ OpenAssetIO extensions for use in Media Creation
 
 Included are several well-known Traits and Specifications for use in
 OpenAssetIO hosts and managers. For more information on this mechanism,
-see the [OpenAssetIO docs](https://thefoundryvisionmongers.github.io/OpenAssetIO/).
+see the [OpenAssetIO docs](https://github.com/OpenAssetIO/OpenAssetIO).
```

Comment:
[fix] Think this is meant to point to the built doxygen docs rather than the GitHub repo
```suggestion
see the [OpenAssetIO docs](https://openassetio.github.io/OpenAssetIO).
```